packages/platform-browser/animations/src/providers.ts

```diff
@@ -18,7 +18,11 @@ import {AnimationRendererFactory} from './animation_renderer';
 @Injectable()
 export class InjectableAnimationEngine extends AnimationEngine implements OnDestroy {
   constructor(
-      @Inject(DOCUMENT) doc: any, driver: AnimationDriver, normalizer: AnimationStyleNormalizer) {
+      @Inject(DOCUMENT) doc: any, driver: AnimationDriver, normalizer: AnimationStyleNormalizer,
```

Comment:
For readability and consistency, we should have line breaks for each of these params now. It's a bit confusing otherwise with the one param on a new line.
```suggestion
      @Inject(DOCUMENT) doc: any,
      driver: AnimationDriver,
      normalizer: AnimationStyleNormalizer,
```

Reply:
I'm running `ng-dev format changed` locally and it still formats it as follows:
```ts
constructor(
      @Inject(DOCUMENT) doc: any, driver: AnimationDriver, normalizer: AnimationStyleNormalizer,
      appRef: ApplicationRef) {
```